Create ui-c8y-1021-22-39-removing-all-translations-from-translation-editor-should-work-8398.md (#2983)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-22-39-removing-all-translations-from-translation-editor-should-work-8398.md

---
date: ""
title: removing all translations from translation editor should work (#8398)
product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62449
version: 1021.22.39
---

> removing all translations from translation editor should work (#8398)

Reviewer: ```suggestion
In the translation editor of the Cumulocity IoT platform, users were previously unable to remove all translations for a specific key. This issue has now been resolved. With this fix, users can successfully remove all translations associated with a key in the translation editor. This change ensures a more consistent and reliable user experience when managing translations within the Cumulocity IoT platform.
```

> In the translation editor of the Cumulocity IoT platform, users were previously unable to remove all translations for a specific key. This issue has now been resolved. With this fix, users can successfully remove all translations associated with a key in the translation editor. This change ensures a more consistent and reliable user experience when managing translations within the Cumulocity IoT platform.

Reviewer 2: ```suggestion
In the translation editor on the **Localization** page, users were previously unable to remove all translations for a specific key. This issue has now been resolved. With this fix, users can successfully remove all translations associated with a key in the translation editor.
```
